lldb/test/Shell/Disassemble/TestDisassembleRISCVXqciInstrutions.test

```diff
@@ -0,0 +1,10 @@
+# REQUIRES: target=riscv32-{{.*}}
+
+# RUN: %lldb -b -o "disassemble -b -n main" %p/Inputs/riscv_xqci.out | FileCheck %s
+
+# CHECK: 051f 0058 0000  qc.e.li a0, 0x58
+# CHECK: 1516              qc.extu a0, a0, 0x6, 0x0
```

Comment:
Maybe the alignment here is due to the way we print tabs and spaces but I think you can make the mnemonics line up and FileCheck will be fine with it.

If not, it's ok as is, it just looks weird to me.

Reply:
Also CHECK-NOT to make sure you don't get unknown here. I know in theory it shouldn't be a problem but it'll prevent us printing it twice or whatever.